core/vibe/core/drivers/libevent2.d

```diff
@@ -1086,6 +1103,34 @@ final class Libevent2UDPConnection : UDPConnection {
 		}
 	}
 
+	void addMembership(string group)
```

Comment:
At least here, the parameter should be of type `ref NetworkAddress`, so that the name resolution happens outside, and the driver function code can focus on the core functionality. But for convenience, `UDPConnection` could have a second `final` overload that accepts a string instead.